data/assets.gresource.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <gresources>
-  <gresource prefix="/com/github/akiraux/akira/partials">
-    <file alias="variables.css" compressed="true">partials/variables.css</file>
-    <file alias="layout.css" compressed="true">partials/layout.css</file>
-    <file alias="artboard.css" compressed="true">partials/artboard.css</file>
-    <file alias="layer.css" compressed="true">partials/layer.css</file>
+  <gresource prefix="/com/github/akiraux/akira/css">
+    <file alias="variables.css" compressed="true">css/variables.css</file>
+    <file alias="layout.css" compressed="true">compressed/layout.css</file>
```

Comment:
A little typo here, I think it should be `css/layout.css`

Reply:
Fixed